docs/regressions-dl19-doc.md

```diff
@@ -0,0 +1,114 @@
+# Anserini: Regressions for [DL19 (Document)](https://github.com/microsoft/TREC-2019-Deep-Learning)
+
+This page documents experiments, integrated into Anserini's regression testing framework, for the TREC 2019 Deep Learning Track (Document Ranking Task) on the MS MARCO document collection using dense (i.e., pooled) relevance judgments from NIST.
```

Comment:
Maybe replace the words "pooled" and "dense" with something easier to understand for a newcomer in IR? Perhaps just say "multiple relevant documents per query".